packages/ui-components/src/__tests__/TokenIOInput.test.ts

```
@@ -2,11 +2,14 @@ import { render, fireEvent, waitFor } from '@testing-library/svelte';
 import { describe, it, expect, vi, beforeEach, type Mock } from 'vitest';
 import TokenIOInput from '../lib/components/deployment/TokenIOInput.svelte';
 import type { ComponentProps } from 'svelte';
-
+import { DotrainOrderGui } from '@rainlanguage/orderbook/js_api';
 export type TokenIOInputComponentProps = ComponentProps<TokenIOInput>;
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)